rllib/agents/ddpg/apex.py

```diff
@@ -16,7 +16,7 @@
         "n_step": 3,
         "num_gpus": 0,
         "num_workers": 32,
-        "buffer_size": 2000000,
+        "buffer_size": 2000001,
```

Comment:
Just curious - what would happen if we don't make the batch size change?

Reply:
Nothing. I honestly have no idea how this got in here.

Reply:
Removed